src/containers/forms/PractitionerForm/UserIdSelect/tests/index.test.tsx

```diff
@@ -22,6 +22,7 @@ describe('src/*/forms/userIdSelect', () => {
       serviceClass: OpenSRPService,
     };
     const wrapper = shallow(<UserIdSelect {...props} />);
+    // tslint:disable-next-line
     await new Promise(resolve => new Promise(resolve));
```

Comment:
Can we throw a mock `growl` error into a `.catch()` here so we don't have to ignore the rule?